backend/base/templates/cancelled_organization.html

```diff
@@ -1,250 +1,167 @@
+            </a>
           </div>
-        </main>
-        <footer class="py-4 bg-light mt-auto">
-          <div
-            class="container-fluid px-4 d-flex align-items-center justify-content-center"
-          >
-            <div class="justify-content-between small">
-              <div class="text-muted">Copyright &copy; Your Website 2023</div>
-              <!-- <div>
-                                <a href="{% url 'termsconditions' %}">Privacy Policy</a>
-                                &middot;
-                                <a href="{% url 'termsconditions' %}">Terms &amp; Conditions</a>
-                            </div> -->
+        </div>
+        <div class="sb-sidenav-footer">
+          <div class="small">Logged in as:</div>
+          <div class="user-email">{{ request.user.email }}</div>
+        </div>
+      </nav>
+    </div>
+    
+    <div id="layoutSidenav_content">
+      <main>
+        <div class="container-fluid px-4">
+          <h1 class="header1 mt-4 mb-4">Cancelled Organization List</h1>
+          <div class="card mb-4">
+            <div class="card-header">
+              <i class="fas fa-table me-1"></i>
+              Organization List
+            </div>
+            <div class="card-body">
+              <table id="datatablesSimple">
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Phone Number</th>
+                    <th>Status</th>
+                  </tr>
+                </thead>
+                <tfoot>
+                  <tr>
+                    <th>Name</th>
+                    <th>Phone Number</th>
+                    <th>Status</th>
+                  </tr>
+                </tfoot>
+                <tbody>
+                  {% for organization in organizations %}
+                    {% if organization.status == 4 %}
+                      <tr>
+                        <td>
+                          <a href="{% url 'organization_preview' organization.pk %}">{{ organization.organization_name }}</a>
+                        </td>
+                        <td>{{ organization.phone_number }}</td>
+                        <td>{{ organization.get_status_display }}</td>
+                      </tr>
+                    {% endif %}
+                  {% endfor %}
+                </tbody>
+              </table>
             </div>
           </div>
-        </footer>
-      </div>
+          <a href="{% url 'tenantuser_page' %}" class="btn btn-primary mt-4" data-toggle="tooltip" title="Back" style="border-radius: 50%; height: max-content; width: max-content; margin-left: auto; margin-right: auto; display: block;">
+            <i class="fas fa-arrow-left"></i>
+          </a>
+        </div>
+      </main>
+      <footer class="py-4 bg-light mt-auto">
+        <div class="container-fluid px-4 d-flex align-items-center justify-content-center">
+          <div class="text-muted">Copyright &copy; Your Website 2023</div>

```

Comment:
Still in 2023?

Reply:
Will change now. I was focusing on yellow page issue caused due to indentation problems.